3scaleAdapter/pkg/threescale/threescale.go

```diff
@@ -27,7 +18,15 @@ import (
 	"istio.io/api/mixer/adapter/model/v1beta1"
 	"istio.io/istio/mixer/pkg/status"
 	"istio.io/istio/mixer/template/authorization"
+	"istio.io/istio/mixer/template/logentry"
 	"istio.io/istio/pkg/log"
+	"net"
+	"net/http"
+	"net/url"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
```

Comment:
Not sure what has happened here, can you run `goimports` on the project?

Reply:
what's the issue?

Reply:
ok, fixing the order.